docs/pseudo_cells.md

```diff
@@ -0,0 +1,36 @@
+## Pseudo Cells
+
+Pseudo PIPs and site pseudo PIPs are edges in the device graph that route
+through other sites and/or BELs.  Pseudo cells are the expression of the what
```

Comment:
"expression of the what" --> "expression of what"

Reply:
Fixed.